### DISCUSSION TITLE
`nnoremap ' "`  creates double qoutes.

### DISCUSSION THREAD

**Discussion by sourcevault** [NeoVintageous/NeoVintageous#930] [Q&A]
https://github.com/NeoVintageous/NeoVintageous/assets/20347647/9b195b6e-e3c9-4ef9-af51-45e728477c64

```
nnoremap " '
```

I have this line of code in my `rc` file, but its not working exactly as I hoped it would.

It emits `""`, basically the double quote character **twice**.

**Comment by gerardroche** [comment]
Mapping the double quote is tricky because it's an integral part of things like registers. But yeah, that's a bug. There have been issue before relating to this. https://github.com/NeoVintageous/NeoVintageous/issues/98

**Reply by sourcevault** [reply]
Thank you gerardroche for being so responsive 🙏.
